src/main/webapp/jsp/instructorCourseEnrollResult.jsp

```diff
@@ -19,8 +19,8 @@
 </c:set>
 
 <ti:instructorPage pageTitle="TEAMMATES - Instructor" bodyTitle="Enrollment Results for ${data.courseId}" jsIncludes="${jsIncludes}">
-    <div class="alert alert-success">
-        <form name='goBack' action="${data.instructorCourseEnrollLink}" method="post" role="form"> 
```

Comment:
Would it work if you move the form outside the div instead? The alert classes are meant to style divs. I.e. `<form><div>...</div></form>`